start-site/src/main/resources/application.yml

```diff
@@ -712,15 +712,25 @@ initializr:
           mappings:
             - compatibilityRange: "[2.0.0.M1,2.1.0.M1)"
               version: 2.22.4
-            - compatibilityRange: "2.1.0.M1"
+            - compatibilityRange: "[2.1.0.M1,2.2.0.M1)"
               version: 2.24.2
-          description: Apache Camel lets you create the Enterprise Integration Patterns to implement routing and mediation rules a Java based Domain Specific Language via Spring.
+          description: Apache Camel is the Swiss knife of integration.
           groupId: org.apache.camel
           artifactId: camel-spring-boot-starter
           links:
-            - rel: guide
+            - rel: reference
+              href: https://camel.apache.org/spring-boot
+        - name: Apache Camel
```

Comment:
We can't have two separate entries for the same artifact. If the `artifactId` has changed, you can define that as part of the mapping entry (i.e. at the same level you map the version, you can also specify a different `groupId`).

Reply:
Ah okay so maybe we can just specify the new groupId instead